sps30-i2c/sps30.c

```diff
@@ -230,3 +236,47 @@ int16_t sps30_start_manual_fan_cleaning() {
 int16_t sps30_reset() {
     return sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_RESET);
 }
+
+int16_t sps30_sleep() {
+    int16_t ret;
+
+    ret = sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_SLEEP);
+    if (ret)
+        return ret;
+
+    sensirion_sleep_usec(SPS_CMD_DELAY_USEC);
+    return 0;
+}
+
+int16_t sps30_wake_up() {
+    int16_t ret;
+
+    /* wake-up must be sent twice, ignore return values */
+    (void)sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_WAKE_UP);
+    (void)sensirion_i2c_write_cmd(SPS30_I2C_ADDRESS, SPS_CMD_WAKE_UP);
```

Comment:
I'd guess so since it precedes the actual wake-up check.

    /* Give the sensor some time to wake up, then send a third command to
     * confirm that the sensor is now responsive */